darshan-util/pydarshan/benchmarks/benchmarks/dxt_heatmap.py

```diff
@@ -0,0 +1,40 @@
+import os
+import importlib
+
+from darshan.experimental.plots import plot_dxt_heatmap
+# TODO: no good reason pydarshan should have hyphenated module
+# names... for now I hack around it...
+example_logs = importlib.import_module("examples.example-logs")
```

Comment:
we really should just rename this module; to some extent the longer-term plan was to make the logs available in another repo, but their *de facto* availability in the current repo is already being leveraged elsewhere